sonar-helm-for-iac/ast.proto

```diff
@@ -65,13 +66,15 @@ message ActionNode {
   int64 pos = 2;
   // The pipeline in the action.
   optional PipeNode pipe = 3;
+  int64 length = 4;
```

Comment:
It will be good to put `length` next to `pos` here. They have the meaning together and are used in all nodes. As this is in "development" cycle we don't have to care about backward compatibility. Later it can be harder to change.
We should have the same order of fields in Go and Java side.
This apply to next Nodes too.